linalg/operator.hpp

```diff
@@ -91,13 +91,34 @@ class Operator
    /** @brief Action of the transpose operator: `y=A^t(x)`. The default behavior
        in class Operator is to generate an error. */
    virtual void MultTranspose(const Vector &x, Vector &y) const
-   { mfem_error("Operator::MultTranspose() is not overloaded!"); }
+   { mfem_error("Operator::MultTranspose() is not overridden!"); }
+
+   /// y += A(x) (default) or y += a * A(x)
+   virtual void AddMult(const Vector &x, Vector &y, const double a = 1.0) const;
+
+   /// y += A^t(x) (default) or y += a * A^t(x)
+   virtual void AddMultTranspose(const Vector &x, Vector &y,
+                                 const double a = 1.0) const;
+
+   /// Operator application on a matrix: `Y=A(X)`.
+   virtual void Mult(const Array<Vector *> &X, Array<Vector *> &Y) const;
+
+   /// Action of the transpose operator on a matrix: `Y=A^t(X)`.
+   virtual void MultTranspose(const Array<Vector *> &X, Array<Vector *> &Y) const;
```

Comment:
These are just copy paste issues from the other docstrings of `Mult` (see line 88 and 91). If we want to keep them, I'll modify `AddMult` and `AddMultTranspose` to be consistent. Otherwise, I can remove them all.

Note that this change corresponds to #3073 on which this PR is based (see my comment below).

Reply:
Fixed in [e5de8d6](https://github.com/mfem/mfem/pull/3075/commits/e5de8d634b1f91d37516bfaad0c16a09c272840b).